src/Libraries/PythonNodeModels/Properties/Resources.en-US.resx

```diff
@@ -128,10 +128,13 @@
     <comment>Description for Python Script</comment>
   </data>
   <data name="PythonScriptEditorAcceptChangesButton" xml:space="preserve">
-    <value>Accept Changes</value>
+    <value>Save Changes</value>
+  </data>
+  <data name="PythonScriptEditorTestButton" xml:space="preserve">
```

Comment:
Can we simply name this `PythonScriptEditorRunButton` as it is essentially the Run button for the Python script editor.

Reply:
Sure, I changed the names of the buttons and decided not to change the underlying methods to make it clearer that it's a cosmetic change, but in hindsight changing the method names is definitely better for long-term maintenance.